<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmInspectTemplate</class>
 <widget class="QWidget" name="frmInspectTemplate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_btn" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btn_create">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>93</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>新建模板</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>93</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>保存模板</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancel_modification">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>93</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>取消修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>93</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>删除模板</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fixed_parameter">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>150</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_fixed_parameter">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>160</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>固定参数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>模板类别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>模板名称</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_type">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>150</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_read_method">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>读数方式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_other_setting">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>其他设置</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_fixed_output">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>231</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>固定5500A输出，读被检表示值</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_adjust_output">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>240</y>
     <width>231</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>调节5500A输出，读5500A示值</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_other_setting">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>240</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>同时检定直流输出的对应负值</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_btn_row" native="true">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>601</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btn_add_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>添加行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_edit_line">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>编辑行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_move_up">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>9</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>上移</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_move_down">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>下移</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete_line">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>删除行</string>
    </property>
   </widget>
  </widget>
  <widget class="QTreeWidget" name="treeWidget_template">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>381</width>
     <height>661</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_template">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>330</y>
     <width>621</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
